src/BoundaryConditions/fill_halo_regions.jl

```diff
@@ -163,28 +163,22 @@ fill_first(bc1, bc2)               = true
 ##### General fill_halo! kernels
 #####
 
-@kernel function _fill_west_and_east_halo!(c, west_bc, east_bc, offset, loc, grid, args) 
+@kernel function _fill_west_and_east_halo!(c, west_bc, east_bc, loc, grid, args) 
```

Comment:
so I guess this is the part that is moved in the KA PR?
perhaps we wait for the to be merged before we review this?

Reply:
yeah we need that PR, we do not want to depend on a branch